lib/checkother.cpp

```diff
@@ -3341,9 +3335,86 @@ void CheckOther::checkEvaluationOrder()
     }
 }
 
-void CheckOther::unknownEvaluationOrder(const Token* tok)
+void CheckOther::checkEvaluationOrderPost11()
+{
+    return;
+}
+
+void CheckOther::checkEvaluationOrderPost17()
+{
+    logChecker("CheckOther::checkEvaluationOrder");
+    const SymbolDatabase *symbolDatabase = mTokenizer->getSymbolDatabase();
+    for (const Scope * functionScope : symbolDatabase->functionScopes) {
+        for (const Token* tok = functionScope->bodyStart; tok != functionScope->bodyEnd; tok = tok->next()) {
+            if (tok->tokType() != Token::eIncDecOp || !tok->astOperand1())
+                continue;
+            for (const Token *tok2 = tok;; tok2 = tok2->astParent()) {
+                const Token * const parent = tok2->astParent();
+                if (!parent || !parent->isBinaryOp())
+                    break;
+                if (parent->str() == ",") {
```

Comment:
It seems strange to me that you excluded this code:

                if (Token::Match(parent, "%oror%|&&|?|:|;"))
                    break;